### PR TITLE
Generate Maven Artifacts

### PR DESCRIPTION
**Pull request by andreban**
- Modified `androidbrowserhelper/build.gradle` to generate the
   Maven artifacts required for publishing the library.

### CODE REVIEW

File: androidbrowserhelper/build.gradle [hunk 1]
```diff
@@ -61,3 +62,41 @@ dependencies {
...
+                }
+            }
+
+            artifact("$buildDir/outputs/aar/androidbrowserhelper-release.aar")
+
+            groupId 'com.google.androidbrowserhelper'
+            artifactId 'androidbrowserhelper'
+            version '0.1.0-alpha1'
+
+            pom.withXml {
```
Comment (PEConn) [inline androidbrowserhelper/build.gradle]: Could you add a comment over why this is necessary (eg link to the Stack Overflow)?

File: androidbrowserhelper/build.gradle [hunk 2]
```diff
@@ -61,3 +62,41 @@ dependencies {
...
+                def dependenciesNode = asNode().appendNode('dependencies')
+
+                // Iterate over the implementation dependencies (we don't want the test ones), adding a <dependency> node for each
```
Comment (PEConn) [inline androidbrowserhelper/build.gradle]: nit: line wrapping.

**Comment by andreban** [comment]
@PEConn PTAL

File: androidbrowserhelper/build.gradle (hunk 2 above)
Comment (andreban) [reply androidbrowserhelper/build.gradle]: Wrapped the line, but added a link that is large :(

**Pull request merged**